File: LeetCode/Easy/3754-concatenate-non-zero-digits-and-multiply-by-sum-i/try-1.java
/*
 * Problem #3754: Concatenate Non-Zero Digits and Multiply by Sum I
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: java
 * Date: 7/7/2026, 7:30:58 PM
 * Link: https://leetcode.com/problems/concatenate-non-zero-digits-and-multiply-by-sum-i/
 */

class Solution {
    public long sumAndMultiply(int n) {
        String s = Integer.toString(n);
        StringBuilder sb = new StringBuilder();
        for (char c : s.toCharArray()) {
            if (c != '0') sb.append(c);
        }
        long x = sb.length() == 0 ? 0 : Long.parseLong(sb.toString());
        long sum = 0;
        long temp = x;
        while (temp > 0) {
            sum += temp % 10;
            temp /= 10;
        }
        return x * sum;
    }
}
    

File: Codeforces/1/A/try-1.py
/*
 * Problem : Theatre Square
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: Python 3
 * Date: 8/6/2026, 7:41:07 PM
 * Link: https://codeforces.com/contest/1/problem/A
 */

n, m, a = map(int, input().split())

rows = (n + a - 1) // a
cols = (m + a - 1) // a

print(rows * cols)
